scripts/get-changed-urls.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# This script lists URLs changed by the target PR. You can jump to target page quickly by clicking those.
+# Syntax: get-changed-urls.sh <PR number> [<page top URL>]
+# Example: 
+# $ get-changed-urls.sh 28542
+# http://localhost:1313/ja/docs/concepts/configuration/manage-resources-containers
+# http://localhost:1313/ja/docs/concepts/policy/limit-range
+# http://localhost:1313/ja/docs/concepts/policy/resource-quotas
+# http://localhost:1313/ja/docs/reference/command-line-tools-reference/feature-gates
+# http://localhost:1313/ja/docs/setup/best-practices/cluster-large
```

Comment:
We should explain how to make that endpoint get served (ie, look at the README!)

Reply:
Thank you for quick comment.
I'll update this paragraph to include "For make endpoint get served, look at the README" sentence.

Reply:
I've updated. Please see [line 15](https://github.com/kubernetes/website/pull/28934/files#diff-4502a903af4b7501bf7003e76feb34dfda313b3beb02f993f72df5b32078f048R15).